concepts/05 UI Components/List/00 Getting Started with List/08 Delete Items.md

```diff
@@ -0,0 +1,61 @@
+To allow users to delete items from the **List**, set the [allowItemDeleting](/Documentation/ApiReference/UI_Components/dxList/Configuration/#allowItemDeleting) property to **true**. The **List** supports several deletion modes which determine how users delete items. Use the [itemDeleteMode](/Documentation/ApiReference/UI_Components/dxList/Configuration/#itemDeleteMode) property to define the most suitable deletion mode in your case.
```

Comment:
The second sentence contains two parts separate by "which" (comma was missing btw), and that's a sign that something might be wrong - either you need to split it into two sentences... Or maybe you should've used "that" instead. In any event "deletion mode" and "how users delete items" are somewhat similar and you can combine them into one expression to avoid the double "delete/deletion" reference. Then you refer to "deletion mode" again in the third sentence. So this all can be optimized to just two sentences instead of three.  

```suggestion
To allow users to delete items from the **List**, set the [allowItemDeleting](/Documentation/ApiReference/UI_Components/dxList/Configuration/#allowItemDeleting) property to **true**. Use the [itemDeleteMode](/Documentation/ApiReference/UI_Components/dxList/Configuration/#itemDeleteMode) property to select the UI elements and/or user actions that remove items.
```